### PR TITLE
Add Synology Surveillance Station Client v1.0.3-0214

### PR DESCRIPTION
**Pull request by tfeldmann**

### CODE REVIEW

File: Casks/synology-surveillance-station-client.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'synology-surveillance-station-client' do
+  version '1.0.3-0214'
+  sha256 '8a9ed4a817307307a3640a2e9bfc34e95a65aa929eef87b7e0f1f51cdd97eece'
+
+  url "https://global.download.synology.com/download/Tools/SurveillanceStationClient/#{version}/Mac/Synology%20Surveillance%20Station%20Client-#{version}.dmg"
+  name 'Synology Surveillance Station Client'
+  homepage 'https://www.synology.com/'
```
Comment (vitorgalvao) [inline Casks/synology-surveillance-station-client.rb]: Isn’t there a more specific URL?

**Comment by vitorgalvao** [comment]
Thank you and welcome. Next time, please fill the template and don’t delete it. It’s there for a reason.

File: Casks/synology-surveillance-station-client.rb (hunk 1 above)
Comment (tfeldmann) [inline Casks/synology-surveillance-station-client.rb]: This one: [https://www.synology.com/surveillance](https://www.synology.com/surveillance)
I took the same url as in casks synology-assistant, synology-cloud-station-drive, etc

**Comment by tfeldmann** [comment]
Oops. Sorry for the template.

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
I did not name this cask `surveillance-station-client` because all other synology applications in homebrew-cask are named `synology-{appname}`. Check `$ brew cask search synology`.

- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

**Comment by vitorgalvao** [comment]
@tfeldmann If the other Synology casks have more specific homepage URLs, mind submitting PRs to fix them?

**Comment by vitorgalvao** [comment]
Please resubmit to [caskroom/drivers](https://github.com/caskroom/homebrew-drivers). I’ll explain why [in the other issue](https://github.com/caskroom/homebrew-drivers/issues/97).

**Pull request closed without merging**